llvm-bazel/llvm-project-overlay/mlir/BUILD

```diff
@@ -4530,26 +4552,43 @@ gentbl(
 )
 
 genlinalg(
-    name = "LinalgNamedStructuredOpsIncGen",
+    name = "LinalgNamedStructuredOpsTcIncGen",
     src = "include/mlir/Dialect/Linalg/IR/LinalgNamedStructuredOpsSpec.tc",
     linalg_outs = [
         (
-            "-gen-impl",
-            "include/mlir/Dialect/Linalg/IR/LinalgNamedStructuredOps.cpp.inc",
+            "-gen-impl -o=$@",
```

Comment:
Looks like there were also changes to linalggen.bzl to not include adding the -o itself

Reply:
Urgh and I'll have to investigate more deeply what's going on here...